tests/storage/app.py

```diff
@@ -30,8 +33,8 @@ def __init__(self, session):
         self.app = flask.Flask("Ert http api")
         self.app.add_url_rule('/ensembles', 'ensembles', self.ensembles)
         self.app.add_url_rule('/ensembles/<ensemble_id>', 'ensemble', self.ensemble_by_id)
-        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations', 'realizations', self.realizations)
-        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations/<realization_id>', 'realization', self.realization_by_id)
+        #self.app.add_url_rule('/ensembles/<ensemble_id>/realizations', 'realizations', self.realizations)
```

Comment:
Would exhausting this functionality include something along the lines of:

```class FlaskWrapper:
    def __init__(self, session):
        self.app = flask.Flask("Ert http api")
        self.app.add_url_rule('/ensembles', 'ensembles', self.ensembles)
        self.app.add_url_rule('/ensembles/<ensemble_id>', 'ensemble', self.ensemble_by_id)
        self.app.add_url_rule('/ensembles/<ensemble_id>/observations', 'observations', self.observations)
        self.app.add_url_rule('/ensembles/<ensemble_id>/observations/<observation_id>', 'observation', self.observation_by_id)
        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations', 'realizations', self.realizations)
        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations/<realization_id>', 'realization', self.realization_by_id)
        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations/<realization_id>/responses', 'responses', self.responses)
        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations/<realization_id>/responses/<response_id>', 'response', self.response_by_id)
        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations/<realization_id>/parameters', 'parameters', self.parameters)
        self.app.add_url_rule('/ensembles/<ensemble_id>/realizations/<realization_id>/parameters/<parameter_id>', 'parameter', self.parameter_by_id)```
??